crm_project_task/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    crm_default_project_id = fields.Many2one("project.project")
```

Comment:
What do you think about it @rafaelbn?
This could be good improvement

Reply:
> Shouldn't this be per crm.team?

It could be and improvement to add but not to replace I mean:

1. As is is OK, same project for all the teams
2. @yajo proposal could be to improve the module for the case that each crm.team should use a different project. So if project is set in crm.team, this override the general one.

I agree "2" could be an improvement, not requested per any client. So in this moment my opinion is to add this improvement @EmilioPascual to the section ROADMAP and not make it until we have a real request.

I'm going to add @yajo (I didn't want to open a big melon 😄 )

- In OCA many modules need to link o create crm.lead with project, each module add the relation.. example: 

https://github.com/OCA/timesheet/blob/15.0/crm_timesheet/models/crm_lead.py#L13

- The best OCA way should be have a base_crm_project module to make this relation in common to other modules like crm_project_task, crm_timesheet, etc...

- And improvement for this module o other, soon, will be the availability to create a new project from crm.lead

- What happed if we want to use both modules separated?

Don't miss: https://github.com/OCA/crm/issues/481

Conclusion: as you see we can think and spend here many hours so in my opinion maintain as is and let see for Odoo 16 or 17 improvements

Reply:
> should be have a base_crm_project module to make this relation in common

In this module, instead of creating a relationship with a field, @EmilioPascual logs a message on both sides of the operation (lead and task). This message links one with the other. IMHO it's enough, so we don't need a m2o field.

In `crm_timesheet`, the lead is linked with a project, not with a task AFAICS, so it'd be a different case anyways.